### DISCUSSION TITLE
Create styles for typography

### DISCUSSION THREAD

**Discussion by Camillep21** [six7/figma-tokens#1509] [Q&A] [answered]
Hello ☀️
I'm trying to create styles from tokens studio plugin to Figma style. When I click on Styles/Create Styles nothing happens.  How can I do ?
 
<img width="250" alt="Capture d’écran 2023-01-03 à 14 42 01" src="https://user-images.githubusercontent.com/121874731/210368743-0533fdee-1258-413f-bc3b-f3e2df7c09be.png">

<img width="1188" alt="Capture d’écran 2023-01-03 à 14 41 07" src="https://user-images.githubusercontent.com/121874731/210368593-e385b347-3734-4de0-bc7b-2c5da2d40f55.png">

Thanks :)

**Comment by six7** [comment, marked as answer]
Click the checkbox right of the `global` token set. It currently shows a dot, which means `Treat as source`. Meaning it will ignore that set and only use it to resolve references. Once its `active` (checked like the `light` set below), it should work.

**Reply by Camillep21** [reply]
Thanks
